test/images/agnhost/webhook/pods.go

```diff
@@ -93,11 +104,9 @@ func mutatePods(ar v1.AdmissionReview) *v1.AdmissionResponse {
 	}
 	reviewResponse := v1.AdmissionResponse{}
 	reviewResponse.Allowed = true
-	if pod.Name == "webhook-to-be-mutated" {
-		reviewResponse.Patch = []byte(podsInitContainerPatch)
-		pt := v1.PatchTypeJSONPatch
-		reviewResponse.PatchType = &pt
-	}
+	reviewResponse.Patch = []byte(patch)
```

Comment:
This change makes us patch all pods, which is a dramatic change in behavior. Why is this desired?

Reply:
TBH, I thought about making this function more generic.
I think checking pod label would be ok. Using only name doesn't allow to apply webhook to pods created by a deployment - this webhook can be used only in specific test case.

Reply:
The image is intended to support specific test cases, which currently create named pods directly. If we have a scenario that needs to inject containers to pods created from deployments, also matching on a label could be reasonable.